modules/bitcoinj/app/src/main/kotlin/bip32/master_key_gen/Wrapper.kt

```diff
@@ -0,0 +1,32 @@
+package bip32.master_key_gen
```

Comment:
I think we can make the package name more generic since the wrapper can have different targets unrelated to bip32. Btw, it's a good idea to do the same in the lightningkmp in another PR. I'll open an issue for that.

src/main/kotlin/bip32/master_key_gen -> src/main/kotlin/bitcoinj

```suggestion
package bitcoinj
```